lib/Field/FieldNew.js

```diff
@@ -0,0 +1,47 @@
+import React, { createContext } from 'react';
+import { bool, node, oneOf, string } from 'prop-types';
+import cx from 'classnames';
+import { FieldContent } from './FieldContent';
+import { FieldMeta } from './FieldMeta';
+import { FieldLabel } from './FieldLabel';
+import { FieldInstructions } from './FieldInstructions';
+
+const InStructureEditModeContext = createContext({
```

Comment:
Nice use of context here 👍

Reply:
This raises another question, should the editing and viewing version of a field be the same component, of should we have `Field` and `EditableField` components with a shared base.

If we were going from the ground up, I imagine we would go with the latter. We like the idea of components having single responsibilities, and when we start to see `if` statements switching a component between modes, that is usually a sign that it may need splitting.

That however is a job for another day, as the current version is already a vast improvement 💯